components/interactive_dialog/dialog_element/dialog_element.test.tsx

```diff
@@ -33,6 +33,22 @@ describe('components/interactive_dialog/DialogElement', () => {
             </React.Fragment>
         ),
     };
+    it('subtype textarea', () => {
+        const wrapper = shallow(
+            <DialogElement
+                {...baseDialogProps}
+                subtype='textarea'
+            />,
+        );
+        expect(wrapper.matchesElement(
+            <TextSetting
+                {...baseTextSettingProps}
+                resizable={true}
+                type='input'
+            />,
+        )).toEqual(true);
```

Comment:
> @mickmister there is still some problem in tests ?

@jatinmark Yes it looks like the new test is failing https://app.circleci.com/pipelines/github/mattermost/mattermost-webapp/61355/workflows/352b773e-9c7a-42f4-9d06-69ec77e25530/jobs/419137.

You can run tests locally to see if these works. I personally use https://marketplace.visualstudio.com/items?itemName=firsttris.vscode-jest-runner to run individual tests locally.

I'm thinking this is correct:

```suggestion
        expect(wrapper.matchesElement(
            <TextSetting
                {...baseTextSettingProps}
                resizable={true}
                type='textarea'
            />,
        )).toEqual(true);
```